app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -211,25 +226,50 @@ private void showFeed() {
 
         txtvDescription.setText(description);
 
-        if (!TextUtils.isEmpty(feed.getAuthor())) {
-            txtvAuthor.setText(feed.getAuthor());
-            txtvAuthorHeader.setText(feed.getAuthor());
-        } else {
-            lblAuthor.setVisibility(View.GONE);
-            txtvAuthor.setVisibility(View.GONE);
-        }
-        if (!TextUtils.isEmpty(feed.getLanguage())) {
-            txtvLanguage.setText(LangUtils.getLanguageString(feed.getLanguage()));
-        } else {
-            lblLanguage.setVisibility(View.GONE);
-            txtvLanguage.setVisibility(View.GONE);
-        }
         txtvUrl.setText(feed.getDownload_url() + " {fa-paperclip}");
         Iconify.addIcons(txtvUrl);
-
         getActivity().invalidateOptionsMenu();
     }
 
+    private void loadStatistics() {
+        if (disposable != null) {
+            disposable.dispose();
+        }
+
+        disposable =
+                Observable.fromCallable(() -> {
+                    List<StatisticsItem> statisticsData = DBReader.getStatistics();
+
+                    for (StatisticsItem statisticsItem : statisticsData) {
+                        if (statisticsItem.feed.getId() == feed.getId()) {
+                            podcastStatistics = statisticsItem;
+                            break;
```

Comment:
You can directly `return statisticsItem` here and use it below (`result`) to avoid storing in a variable